salt/modules/virt.py

```diff
@@ -1955,6 +2081,70 @@ def init(
                 'memnodes': {0: {'mode': 'strict', 'nodeset': 1}, 1: {'mode': 'preferred', 'nodeset': 2}}
             }
 
+    .. _init-cpu-def:
+
+    .. rubric:: cpu parameters definition
+
+    The cpu parameters dictionary can contain the following properties:
+
+    cpuset
+        a comma-separated list of physical CPU numbers that domain process and virtual CPUs can be pinned to by default
```

Comment:
May be you should provide examples here. It also can handle ranges and excluded CPUs like `1-4,^3,6`